modules/ROOT/pages/endpoints.adoc

```diff
@@ -1,13 +1,13 @@
 = Endpoints
 
-Endpoints define the transport protocol configurations necessary to receive or send messages with external trading partners or internal backend applications.
+Endpoints define the required transport protocol configurations to receive or send messages with external trading partners or internal backend applications.
```

Comment:
I actually prefer the flow of the first version, or at least the position of the word, if not the word:

Endpoints define configurations 
necessary to receive or send messages

The endpoints themselves are not as necessary as what the user wants: to receive and send messages.

But this is a nit.

Reply:
Returned to original wording.